polkadot/runtime/common/src/slots/mod.rs

```diff
@@ -192,11 +213,10 @@ pub mod pallet {
 
 			// Refund any deposits for these leases
 			for (who, deposit) in deposits {
-				let err_amount = T::Currency::unreserve(&who, deposit);
-				debug_assert!(err_amount.is_zero());
```

Comment:
Why remove the debug_assert?

Reply:
Its not removed but moved inside the `fn unreserve` as defensive_assert.